src/ansys/mapdl/core/_commands/database/picking.py

```diff
@@ -20,147 +20,137 @@
 
-        Caution:: : For a given entity type, a list containing an ITEM that is
-        larger than the maximum defined entity, could deplete the system memory
-        and produce unpredictable results.
+            * ``3`` - Keypoint numbers
 
-        This command is valid in any processor.
-        """
-        command = "FITEM,%s,%s,%s,%s" % (
-            str(nfield),
-            str(item),
-            str(itemy),
-            str(itemz),
-        )
-        return self.run(command, **kwargs)
+            * ``4`` - Line numbers
 
-    def flst(self, nfield="", narg="", type_="", otype="", leng="", **kwargs):
-        """Specifies data required for a picking operation (GUI).
+            * ``5`` - Area numbers
 
-        APDL Command: FLST
+            * ``6`` - Volume numbers
 
-        Parameters
-        ----------
-        nfield
-            Field number on the command which uses the picking data.  (Count
-            the command name as a field, so that a 2 indicates the first
-            command argument, 3 for the second command argument, etc.)  The
-            corresponding field on the command will have a P51X label.
+            * ``7`` - Trace points
 
-        narg
-            Number of items in the picked list.
+            * ``8`` - Coordinate locations (in Global Cartesian coordinates)
 
-        type\\_
-            Type of items picked:
+            * ``9`` - Screen picks (in X, Y screen coordinates (-1 to 1))
 
-            1
-                Node numbers
+        otype : str
+            Data order:
 
-            2
-                Element numbers
+            * ``NOOR`` - Data is not ordered (default).
 
-            3
-                Keypoint numbers
+            * ``ORDER`` - Data is in an ordered list (such as for the :ref:`e`,P51X and :ref:`a`,P51X commands, in which the
+              order of the data items is significant for the picking operation).
 
-            4
-                Line numbers
+        leng : str
+            Length of number of items describing the list (should equal ``NARG`` if ``Otype`` = NOOR;
+            default).
 
-            5
-                Area numbers
+        Notes
+        -----
+
+        .. _FLST_notes:
 
-            6
-                Volume numbers
+        Specifies data required for the :ref:`fitem` command during a picking operation. This is a command
+        generated by the GUI and will appear in the log file ( :file:`Jobname.LOG` ) if graphical picking is
+        used. This command is not intended to be typed in directly in a session (although it can be included
+        in an input file for batch input or for use with :ref:`input` ).
 
-            7
-                Trace points
+        On the log file, :ref:`flst` is always followed by one or more :ref:`fitem` commands which, in turn,
+        are followed by the Mechanical APDL command containing a P51X label in one of its fields. Do not
+        edit this
+        set of commands.
```

Comment:
weird break

Reply:
Thanks, adding it to the https://github.com/ansys/pyconverter-xml2py/issues/395 issue thread.